### DISCUSSION TITLE
Ubuntu version

### DISCUSSION THREAD

**Discussion by stafbulp** [RagingTiger/docker-cups-airprint#12] [Q&A] [answered]
Hi! Is there a reason for using Ubuntu Xenial as base for this image? I am looking for a way to upgrade cups to a later version, so would changing to e.g. Ubuntu Groovy work?

**Comment by RagingTiger** [comment, marked as answer]
Please see issue #9 as it deals with this quite extensively. Basically there are some issues with **ARM** and the more recent **Ubuntu** image versions. Still I created a **dev** branch that deals with the migration to **Python 3** and **Ubuntu 20.04**.

You can pull down the [dev](https://github.com/RagingTiger/docker-cups-airprint/tree/dev) branch and build it locally. It has not been extensively tested but should work fine 😅

**Reply by stafbulp** [reply]
Great, thanks! I'll give that a try.

**Reply by RagingTiger** [reply]
Let me know how it goes. Would be good to get some actual user feedback on the [dev](https://github.com/RagingTiger/docker-cups-airprint/tree/dev) branch 😄

**Comment by RagingTiger** [comment]
Update: There is now a `tigerj/cups-airprint:dev` docker image built from the [dev](https://github.com/RagingTiger/docker-cups-airprint/tree/dev) branch. See this issue for more information: https://github.com/RagingTiger/docker-cups-airprint/issues/9#issuecomment-824044960

**Reply by stafbulp** [reply]
I just tried the dev docker image and it seems to be working fine! Many thanks!

**Reply by RagingTiger** [reply]
Awesome. If there are any issues, open an issue and we'll troubleshoot it 😄
